lib/engine/game/g_18_mex.rb

```diff
@@ -283,7 +283,7 @@ def event_ndm_merger!
         end
 
         @mergable_candidates = mergable_corporations
-        @log << "Merge candidates: #{@mergable_candidates.map(&:name)}" if @mergable_candidates.any?
+        @log << "Merge candidates: #{present_mergable_candidates(@mergable_candidates)}" if @mergable_candidates.any?
```

Comment:
Needs the 'e' Mergeable

Otherwise fine

Reply:
I have pushed a commit that change all 21 occurrences of mergable to mergeable.